src/resources/js/commerce/gateway/toggler.js

```diff
@@ -141,7 +148,7 @@ tribe.tickets.commerce.gateway = tribe.tickets.commerce.gateway || {};
 			obj.toggler.hideGateway( gateway );
 		});
 		obj.toggler.toggles.default.addClass(obj.toggler.classes.toggleOpen).hide().attr( 'aria-hidden', 'true' );
-		obj.toggler.toggles.additional.removeClass(obj.toggler.classes.toggleOpen);
+		obj.toggler.toggles.additional.removeClass(obj.toggler.classes.toggleOpen).attr( 'aria-selected', 'false' );
```

Comment:
If I'm not mistaken we're applying this to the `containers` of all the additional gateways which are not the default, no?

The aria-selected should be applied to the "opener" and not the container.

Reply:
This is actually for the toggle button itself. The toggle button is the `opener`